src/index.rst

```diff
@@ -111,17 +122,20 @@ indicate hard dependencies; dashed links indicate optional dependencies.
 Other Tools Application Developers May Find Useful
 ======================================================
 
-Axom developers support other tools that can be used by software 
-projects independent of the Axom. These include:
+The Axom team develops and supports other software tools that are useful
+for software projects independent of the Axom. These include:
 
-  *  `BLT <https://github.com/LLNL/blt>`_: CMake-based build system developed by the Axom team to simplify CMake usage and development tool integration
-  *  `Shroud <https://github.com/LLNL/shroud>`_: Generator for native C and Fortran APIs from C++ code
-  *  `Conduit <https://lc.llnl.gov/confluence/display/CON/Conduit+Home>`_: Library for describing and managing in-memory data structures
+  *  `BLT <https://github.com/LLNL/blt>`_ CMake-based build system developed by the Axom team to simplify CMake usage and development tool integration
+  *  `Shroud <https://github.com/LLNL/shroud>`_ Generator for Fortran and Python interfaces to C or C++ libraries
```

Comment:
I notice that this description is taken from the Shroud page, so it is correct. I was just never sure how to mention that it will generate the native C wrappers for C++ code without being too verbose. I only mention this because the page mentions "native interfaces in C and Fortran" above. "Generator for Fortran, C and Python interfaces to C or C++ libraries" feels odd because there is no need for a C interface to a C library. Not really suggesting a change, but I'm open to suggestions for the Shroud github page description.

Reply:
Yeah, that is a bit confusing. I'll attempt to clarify the point to reduce confusion.